### DISCUSSION TITLE
a way to make a layer-shell component not have reserved space

### DISCUSSION THREAD

**Discussion by nekopsykose** [YaLTeR/niri#543] [General]
(or at least that's how i'm wording it)

currently, when doing maximize-column with e.g. a waybar on the right, it reserves space that takes away from the column width: ![](https://img.ayaya.dev/B1mPUpUcDOSy)

usually this is fine (and of course perfectly intended given that you'd normally have a full bar there), but in my case since i have what amounts to the world's smallest widget that i don't care about overlapping the column/window contents itself, is there a way to maximize-column to the entire screen and not have any space reserved for waybar?

i had a look through the config but didn't find a way to override anything related to reserved space for layer-shell (and e.g. struts right 0 doesn't do anything because it's additive)

**Comment by YaLTeR** [comment]
There's a Waybar setting that makes it not request an exclusive zone from layer shell. Then it won't subtract from the working area.

**Reply by nekopsykose** [reply]
and so there is! thank you! always hard to find this stuff without knowing the terms...

for whoever finds this, it's `"exclusive": false`
